### ISSUE TITLE
[Backend] When in drum mode, I can click on a pad and get 1 audio file played

### ISSUE THREAD

**Issue by malparty** [malparty/nimbleat#7]
## Acceptance criteria

- Make a click on a pad plays a single audio file (the same file for each pad)

> We will add different sounds in another task

> You can store the audio file in the repository — please just use a short file (e.g. a single clap). Let's chose a file that starts immediately (i.e. let's avoid files were the clap occurs only after some time)
